doc/api/querystring.md

```diff
@@ -118,8 +118,9 @@ The `querystring.stringify()` method produces a URL query string from a
 given `obj` by iterating through the object's "own properties".
 
 It serializes the following types of values passed in `obj`:
-{string|number|boolean|string[]|number[]|boolean[]}
-Any other input values will be coerced to empty strings.
+{string|number|boolean|string[]|number[]|boolean[]}.
+Any other input values (including `null` and `undefined`) will be coerced to
```

Comment:
To be honest I think the original is better. `null` and `undefined` already fall in the "any other input" set so this addition seems redundant to me.